### PR TITLE
Mergeboxplots

### PR DESCRIPTION
**Pull request by msvinaykumar**

### CODE REVIEW

**Comment by dinogun** [comment]
@msvinaykumar Please fix the conflcits

**Comment by msvinaykumar** [comment]
> @msvinaykumar Please fix the conflcits

done

**Comment by dinogun** [comment]
@msvinaykumar more conflicts as PR #1081 was merged. Please fix

**Comment by msvinaykumar** [comment]
> @msvinaykumar more conflicts as PR #1081 was merged. Please fix

Done

File: src/main/java/com/autotune/analyzer/kruizeObject/KruizeObject.java [hunk 1]
```diff
@@ -112,6 +111,22 @@ public KruizeObject() {
...
+        // TODO: add CRD for terms
+        terms.put(KruizeConstants.JSONKeys.SHORT_TERM, new Terms(KruizeConstants.JSONKeys.SHORT_TERM, KruizeConstants.RecommendationEngineConstants
+                .DurationBasedEngine.DurationAmount.SHORT_TERM_DURATION_DAYS, KruizeConstants.RecommendationEngineConstants
+                .DurationBasedEngine.DurationAmount.SHORT_TERM_DURATION_DAYS_THRESHOLD, 4, 0.25));
+        terms.put(KruizeConstants.JSONKeys.MEDIUM_TERM, new Terms(KruizeConstants.JSONKeys.MEDIUM_TERM, KruizeConstants
+                .RecommendationEngineConstants.DurationBasedEngine.DurationAmount.MEDIUM_TERM_DURATION_DAYS, KruizeConstants
+                .RecommendationEngineConstants.DurationBasedEngine.DurationAmount.MEDIUM_TERM_DURATION_DAYS_THRESHOLD, 7, 1));
+        terms.put(KruizeConstants.JSONKeys.LONG_TERM, new Terms(KruizeConstants.JSONKeys.LONG_TERM, KruizeConstants
+                .RecommendationEngineConstants.DurationBasedEngine.DurationAmount.LONG_TERM_DURATION_DAYS, KruizeConstants
+                .RecommendationEngineConstants.DurationBasedEngine.DurationAmount.LONG_TERM_DURATION_DAYS_THRESHOLD, 15, 1));
```
Comment (dinogun) [inline src/main/java/com/autotune/analyzer/kruizeObject/KruizeObject.java:125]: The values here dont seem to make sense. Can you please add a javadoc for all of these? Also the last number seems to be wrong

**Comment by dinogun** [comment]
Closing as this branch was created on upstream/origin

**Pull request closed without merging**